src/test/java/com/example/produktapi/resources/features/StepDefinition.java

```diff
@@ -124,6 +123,30 @@ public void checkTheQuantityInTheCheckoutButton() throws InterruptedException {
             }
             }
 
+
+
+        @When("the user clicks on the {string} button")
+        public void theUserClicksOnTheButton(String buttonName) {
+            WebDriverWait wait = new WebDriverWait(driver, Duration.ofSeconds (30));
```

Comment:
30 seconds is too long, it should be enough with 2 or 3 seconds

Reply:
yes I can decrease the time to 10 seconds but in 2 and 3 seconds it is hard to find elements.